cuda_bindings/setup.py

```diff
@@ -304,9 +302,9 @@ def finalize_options(self):
 setup(
     version=versioneer.get_version(),
     ext_modules=do_cythonize(extensions),
-    packages=find_packages(include=["cuda.cuda", "cuda.cuda.*", "cuda.cuda.bindings", "cuda.cuda.bindings._bindings", "cuda.cuda.bindings._lib", "cuda.cuda.bindings._lib.cyruntime", "cuda.cuda.bindings._internal", "tests"]),
+    packages=find_packages(include=["cuda", "cuda.*", "cuda.bindings", "cuda.bindings._bindings", "cuda.bindings._lib", "cuda.bindings._lib.cyruntime", "cuda.bindings._internal"]),
     package_data=dict.fromkeys(
-        find_packages(include=["cuda.cuda", "cuda.cuda.*", "cuda.cuda.bindings", "cuda.cuda.bindings._bindings", "cuda.cuda.bindings._lib", "cuda.cuda.bindings._lib.cyruntime", "cuda.cuda.bindings._internal", "tests"]),
+        find_packages(include=["cuda", "cuda.*", "cuda.bindings", "cuda.bindings._bindings", "cuda.bindings._lib", "cuda.bindings._lib.cyruntime", "cuda.bindings._internal"]),
         ["*.pxd", "*.pyx", "*.py", "*.h", "*.cpp"],
     ),
```

Comment:
@vzhurba01 I guess I missed this during review -- were these changes needed as part of the hot fix?

In any case, I think this might have to do with the `package_data` issue that you hit in https://github.com/NVIDIA/cuda-python/issues/215#issuecomment-2458872021. I had a similar observation for `cuda.core`, and the solution was highlighted here: https://github.com/NVIDIA/cuda-python/pull/171#discussion_r1802086675. I think we have two ways:
1. Revert the changes in `setup.py` (given that it built fine in the previous patch release)
2. Apply the same treatment in `cuda.core` here.

Reply:
(1) isn't quite true. While that build did work, the posted Windows packages for 11.8.4 and 12.6.1 are both missing all the wildcard files.

(2) Thanks for the pointer. I tried an iteration of something similar, but with whatever I tried I couldn't quite get it to stick. It wasn't identical to what you have there, so I'll try again as soon as I get the chance! 🙏